kapitan/inputs/jinja2_filters.py

```diff
@@ -47,6 +51,19 @@ def load_jinja2_filters(env):
     env.filters['shuffle'] = randomize_list
 
 
+def load_jinja2_filters_from_file(env, filter_paths):
```

Comment:
In this function, you also need to check whether filter_paths are actual paths/directories or files with os.path.isdir/isfile etc. If you get a `filter_paths` that points to an empty directory, maybe the function should at least warn the user without stopping execution? To prevent mistakes and make debugging easier.

Reply:
In case if user provides path to a directory then should we walk in it and load the all the modules present there ?

Reply:
We can start by taking into account only modules in the provided directory. This is easy to expand later on as well if needed. It should be a one line change + test change.  :+1:

Reply:
As nargs='+' so he may provide mixture of file path and directories. I am thinking to separate the logic and flatten all file paths first and then load them.

Reply:
Hey @uberspot , I have refactored the logic, have a look.